whisper/whisperv6/peer.go

```diff
@@ -127,6 +129,11 @@ func (peer *Peer) handshake() error {
 		}
 	}
 
+	b, err := s.Bool()
+	if b && isLightNode {
+		return fmt.Errorf("peer [%x]: useless peer: two pure light node communication", peer.ID())
```

Comment:
This is an extreme solution; furthermore, it means that light peers are rejected outright. Meaning light nodes can not connect or receive messages, which is a behavior that *at least* should be configurable. Otherwise, it makes no sense to have light nodes in the first place. Also, what would prevent me from lying if I was running a light node?

I would prefer to keep them on a list and let the user decide if they want to starve light nodes or not.

@gluk256 what do you think?